brainscore/benchmarks/Lee2020.py

```diff
@@ -0,0 +1,181 @@
+
+    def __init__(self, metric=ks_similarity, bootstrap_samples=100_000, num_sample_arrays_candidate=10, ceiler_fn=InternalConsistency):
+
+        def load_target_assembly():
+            assembly = make_static(brainscore.get_assembly('dicarlo.MajajHong2015').sel(region='IT'))
+            return assembly[:, np.in1d(assembly.presentation.image_id, list(set(stimulus_set['image_id'])))]
+
+
+        stimulus_set = Rajalingham2019()._target_assembly.stimulus_set
+        assembly = load_target_assembly()
+
+        super(self).__init__(identifier='dicarlo.Lee2020.IT-tissue',
+                             ceiling_func=lambda: ceiler_fn()(assembly.transpose('presentation','neuroid')),
+                             version=0.1,
+                             parent='IT',
+                             bibtex=BIBTEX)
+
+        self._stimulus_set = stimulus_set
+        self._target_assembly = average_repetition(assembly)
+        self._target_statistic_fn = self.create_tissue_response_correlation_target
+        self._metric = metric
+
+        self.bootstrap_samples = bootstrap_samples
+        self.num_sample_arrs = num_sample_arrays_candidate
+        self._arr_size_mm = (np.ptp(self._target_assembly.neuroid.x.data),
+                             np.ptp(self._target_assembly.neuroid.y.data))
+
+    def __call__(self, candidate):
+
+        candidate.start_recording(recording_target='IT', time_bins=[(70,170)])
+        activations = candidate.look_at(self._stimulus_set)
+        activations = make_static(activations)
+
+        pairwise_tissue_corr_candidate = []
+        for window in sample_array_locations(activations.neuroid, self.num_sample_arrs, self._arr_size_mm):
+            pairwise_tissue_corr_candidate.append(sample_response_corr_vs_dist(activations[window],
+                                                              int(self.bootstrap_samples/self.num_sample_arrs)))
+
+        return self.score(np.hstack(pairwise_tissue_corr_candidate))
+
+    def create_tissue_response_correlation_target(self):
+
+        pairwise_tissue_corr_target = []
+        for animal in set(self._target_assembly.neuroid.animal.data):
+            for arr in set(self._target_assembly.neuroid.arr.data):
+
+                sub_assembly = self._target_assembly.sel(animal=animal, arr=arr)
+                bootstrap_samples_sub_assembly = int(self.bootstrap_samples*(sub_assembly.neuroid.size/
+                                                                             self._target_assembly.neuroid.size))
+
+                pairwise_tissue_corr_target.append(sample_response_corr_vs_dist(sub_assembly,
+                                                                                bootstrap_samples_sub_assembly,
+                                                                                self.ceiling))
+
+        return np.hstack(pairwise_tissue_corr_target)
+
+    def score(self, candidate_statistic):

```

Comment:
this entire method (and methods it uses) should actually be a **metric** so that multiple benchmarks can re-use it on different data.

Maybe it would also make sense to move the characterization (or property) functions into a common directory so that they too can be re-used.  But that doesn't have to happen here, it can be a thing in case another benchmark wants to re-use your code.